package.json

```diff
@@ -114,6 +114,7 @@
     "marked": "^15.0.7",
     "mini-css-extract-plugin": "^2.7.2",
     "minimist": "^1.2.5",
+    "node-addon-api": "^8.3.1",
```

Comment:
I needed this to be able to build/run the precommit checks

Reply:
-1 I don't understand what you mean. I assume it's not needed/outdated by now.